docs/docs/model/notebooks/pythonmodel_type_hints_quickstart.ipynb

```diff
@@ -0,0 +1,277 @@
+{
+ "cells": [
```

Comment:
Same for this notebook. Mose cells are not executed. It would be better if users can see the outputs without running the notebook itself.